README.md

```diff
@@ -0,0 +1,123 @@
+# API Specification Version Maven Plugin
+
+The API Specification Version Plugin provides goals to set properties for manifest file OSGI headers and to verify consistence of specification, API and API implementation package names and versions.
```

Comment:
I would change this to mention the document that Bill put together. We should probably have it checked-in the ee4j parent project as a markdown file instead of a wiki page.

Then we can state something like "this plugin helps generating and enforcing compliant spec metadata".

Reply:
Do you have URL od this document?

Reply:
https://wiki.eclipse.org/JakartaEE_Maven_Versioning_Rules

Reply:
Done